catalog/transformer/catalog-transformer-streaming-lib/src/main/java/org/codice/ddf/transformer/xml/streaming/lib/SaxEventToXmlElementConverter.java

```diff
@@ -58,6 +61,11 @@
      */
     private Set<String> namespacesAdded = new HashSet<>();
 
+    /*
+     * Stack of scoped namespace URI to prefix mappings
+     */
+    private Deque<Map<String, String>> scopeOfNamespacesAdded = new ArrayDeque<>();
```

Comment:
Do you mean a Deque isn't threadsafe? Yea that's okay, an instance of this class (SaxEventToXmlElementConverter) will only be hit from one thread.